tests/encrypted_lvm.spec.ts

```diff
@@ -14,37 +15,69 @@ test.describe('The main page', () => {
     });
 
     test('Installation test with encrypted lvm file system', async ({ page }) => {
+        // optional actions done on the page
+        const actions = Object.freeze({
+          setProduct: Symbol("product"),
+          setMainPage: Symbol("main"),
+          done: Symbol("done")
+        });
+
+        // check for multiple texts in parallel, avoid waiting for timeouts
+        let action = await Promise.any([
+          page.getByText("Product selection").waitFor().then(() => actions.setProduct),
+          page.getByText(process.env.PRODUCTNAME).waitFor().then(() => actions.setMainPage),
```

Comment:
we can wait here for the Install button instead, so we don't need specific product and we do this generic, later on we can create some specific test for checking titles and labels everywhere.

Reply:
Updated